packages/react-dom-bindings/src/client/ReactDOMHostConfig.js

```diff
@@ -219,6 +219,16 @@ export function getPublicInstance(instance: Instance): Instance {
   return instance;
 }
 
+export function getNamespace(hostContext: HostContext): string {
+  if (__DEV__) {
+    const hostContextDev: HostContextDev = (hostContext: any);
+    return hostContextDev.namespace;
+  } else {
+    const hostContextProd: HostContextProd = (hostContext: any);
+    return hostContextProd;
+  }
+}
+
```

Comment:
This is so we don't leak the HostContext type out of HostConfig. should get inlined since it is currently only used in 1 place